### PR TITLE
[Ref] UI clean-up and consistent styling across platforms

### PR DESCRIPTION
**Pull request by saifulbkhan**
This branch contains the following changes:
* a consistent theme for all three desktop environments (helps deal with many platform-specific UI issues)
* several minor changes to the UI based on existing issues
* removal of many classes and widgets that were no longer used or were non-functional
* removal of "libpillow" as a dependency (we do not need an HTTP server, at least yet)
* removal of input formats no longer acceptable (peptide databases, RAW files, mzData, etc.)

### CODE REVIEW

**Comment by saifulbkhan** [comment]
@sakshikukreja14 In this branch (and onwards) we are using a submodule. So when you fetch this branch and checkout, make sure to call this:
```bash
git submodule update --init --recursive
```
to compile successfully.

**Comment by sakshikukreja14** [comment]
@saifulbkhan can we not change the heading's color in dialog boxes that appear? With combination of purple's shades and white everywhere, this grey looks somewhat weird.

**Comment by saifulbkhan** [comment]
@sakshikukreja14 If you mean window title-bars, then unfortunately, no. The title-bars for dialogs and windows are drawn by the OS. You can ask Qt to get rid of the OS-drawn title-bar and then draw your own, but then you will have to manage all sorts of thing by yourself (position, size, close, minimize, maximize, etc.), which is too much work. Don't worry, you will get used to it soon enough. :)

File: src/gui/mzroll/forms/peakdetectiondialog.ui [hunk 1]
```diff
@@ -848,40 +886,79 @@
...
+              <property name="value">
+               <double>0.500000000000000</double>
+              </property>
+             </widget>
+            </item>
+            <item row="0" column="0">
+             <widget class="QLabel" name="label_10">
               <property name="text">
-               <string>Minimum Quality</string>
+               <string>Minimum Ppeak intensity</string>
```
Comment (sakshikukreja14) [inline src/gui/mzroll/forms/peakdetectiondialog.ui:922]: Spellings error.

File: src/gui/mzroll/mainwindow.cpp [hunk 2]
```diff
@@ -319,6 +319,11 @@ using namespace mzUtils;
     styleSheet += "QStatusBar QLabel { margin: 4px; }";
     styleSheet = styleSheet.arg(border.name(QColor::HexRgb));
 
+    styleSheet += "QToolTip { background-color: black; }";
+    styleSheet += "QToolTip { color: white; }";
+    styleSheet += "QToolTip { border: 1px solid black; }";
+    styleSheet += "QToolTip { font-size: 16px; }";
```
[the remark this answers is not in the document]
Comment (saifulbkhan) [reply src/gui/mzroll/mainwindow.cpp:324]: Oh. It looked okay on my large monitor. I will reset it to the default application's font-size.

File: src/gui/mzroll/forms/peakdetectiondialog.ui (hunk 1 above)
Comment (saifulbkhan) [reply src/gui/mzroll/forms/peakdetectiondialog.ui:922]: Yes. Will correct.

**Comment by sakshikukreja14** [comment]
Changes looks cool and works fine. @saifulbkhan you can fix spellings error and we can merge it. :)

**Review by sakshikukreja14 — APPROVED** [review]
Approving :)

**Pull request merged**